src/nvme/mi.c

```diff
@@ -715,85 +715,80 @@ int nvme_mi_admin_xfer(nvme_mi_ctrl_t ctrl,
 	return 0;
 }
 
-int nvme_mi_admin_admin_passthru(nvme_mi_ctrl_t ctrl, __u8 opcode, __u8 flags,
-				 __u16 rsvd, __u32 nsid, __u32 cdw2, __u32 cdw3,
-				 __u32 cdw10, __u32 cdw11, __u32 cdw12,
-				 __u32 cdw13, __u32 cdw14, __u32 cdw15,
-				 __u32 data_len, void *data, __u32 metadata_len,
-				 void *metadata, __u32 timeout_ms, __u32 *result)
+int nvme_mi_admin_passthru(nvme_mi_ctrl_t ctrl, struct nvme_passthru_cmd *cmd, __u32 *result)
```

Comment:
We'll end up with two functions with similar names:

* `nvme_mi_admin_passthru`; and
* `nvme_mi_admin_admin_passthru`

\- is there a batter way to distinguish these?